customer-service/src/main/java/com/spaghettininjas/yaposs/controller/CustomerController.java

```diff
@@ -0,0 +1,37 @@
+package com.spaghettininjas.yaposs.controller;
+
+import com.spaghettininjas.yaposs.service.CustomersService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.spaghettininjas.yaposs.repository.entity.Customer;
+import java.util.Arrays;
+
+@RestController
+@RequestMapping("/api/Customers")
+public class CustomerController {
```

Comment:
Some end-points haven't been implemented:

- __`GET`__ `/api/Customers`

  Retrieves a list of Customers. Supports pagination and filtering through query parameters.

- __`PUT`__ `/api/Customers/{id}`

  Updates an existing Customer identified by {id}. The request body should include updated details.

Reply:
The work is still in progress :D

Reply:
addressed in https://github.com/domanteli0/PoSS/pull/4/commits/84ca7b3bbdd41a6b79b83808cfd2337adfb52d07